server/src/main/java/io/spine/server/reflect/EventSubscriberMethod.java

```diff
@@ -57,16 +58,15 @@ private EventSubscriberMethod(Method method) {
     /**
      * Invokes the subscriber method in the passed object.
      */
-    public static void invokeSubscriber(Object target, Message eventMessage, EventContext context) {
+    public static void invokeFor(Object target, Message eventMessage, EventContext context) {
         checkNotNull(target);
         checkNotNull(eventMessage);
         checkNotNull(context);
 
         try {
-            final EventSubscriberMethod method = forMessage(target.getClass(),
-                                                            eventMessage);
+            final EventSubscriberMethod method = getMethod(target.getClass(), eventMessage);
             method.invoke(target, eventMessage, context);
-        } catch (InvocationTargetException e) {
+        } catch (RuntimeException e) {
```

Comment:
Not sure about this. Aren't we catching something we are not suppose to handle on this level? Let's discuss tomorrow in person.